docs/development/releasing/index.md

```diff
@@ -1,42 +1,53 @@
 # Releasing
 
-## Helm Charts
+As prerequisites to release:
+
+- The repository that holds the code for the component/package to release has:
+  - A release commit as last commit in `master` branch
+  - The last commit in master is tagged.
 
-### 1. Build and test locally:
+## Python packages
+
+**1.** Release the package(s)
 
 ```console
-$ reana-dev run-ci \
-    --admin-email john.doe@example.org \
-    --admin-password 123456 \
-    --component r-d-r-roofit \
-    --exclude-components=r-ui,r-m-broker
+$ reana-dev release-pypi -c <package-name>
```

Comment:
If somebody would copy paste, using syntax like "<foo>" will lead to shell redirections... Better to use other more directly followable example.

E.g. if the example emulates releasing new `r-server` package, you can use:

```console
$ cd ~/mysrc/reana-server
$ reana-dev docker-build -c . --no-cache
```

Reply:
(Actually, the `--no-cache` step should have been done before, while testing all components together in production conditions, otherwise some docker layer caching problem could pop up at this (late) stage without it being fully tested...)